test/ModalSpec.js

```diff
@@ -220,6 +220,24 @@ describe('Modal', function () {
     });
   });
 
```

Comment:
Can you add a test for the case when a `closeButton` is requested but no custom element is provided? (Verify that it displays the default close button)